lib/Sema/CSSimplify.cpp

```diff
@@ -2896,6 +2896,30 @@ getArgumentLabels(ConstraintSystem &cs, ConstraintLocatorBuilder locator) {
   return known->second;
 }
 
+
+// Return true if the specified type conforms to the DynamicLookupMemberProtocol
+// or if it is a protocol that inherits from it.
+static bool containsOrConformsToDLMP(Type ty, DeclContext *DC,
+                                     Identifier name, TypeChecker &TC) {
+  auto DMLP =
+    TC.Context.getProtocol(KnownProtocolKind::DynamicMemberLookupProtocol);
+
+  auto conformance =
+    TC.conformsToProtocol(ty, DMLP, DC, ConformanceCheckFlags::InExpression);
+  
+  // If this type doesn't conform to it, check to see if this is an existential
+  // that contains it.
+  if (!conformance)
+    conformance =
+      TC.containsProtocol(ty, DMLP, DC, ConformanceCheckFlags::InExpression);
```

Comment:
You should check but I think containsProtocol() is a superset of conformsToProtocol() so you only need the former here

Reply:
Looks like it!  Thanks